pydantic_ai_slim/pydantic_ai/realtime/_openai_protocol.py

```
@@ -0,0 +1,360 @@
+
+
+@dataclass
+class ServerVAD:
+    """Server-side voice activity detection — the default turn-taking mode.
+
+    The server detects when the user starts and stops speaking and (by default) commits the audio
+    and triggers a response automatically. Unset fields fall back to the provider defaults.
+    """
+
+    threshold: float | None = None
+    """Activation threshold (0.0–1.0). Higher requires louder audio; better in noisy environments."""
```

Ruff (0.15.20)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)